Makefile

```diff
@@ -316,19 +316,58 @@ stat: 		## display processes with PID(s) of gunicorn instance(s) running the app
 ## --- Docker targets --- ##
 
 .PHONY: docker-info
-docker-info:	## tag version of docker image for build/push
-	@echo "Docker image will be built, tagged and pushed as:"
-	@echo "$(DOCKER_TAG)"
+docker-info:		## obtain docker image information
+	@echo "Docker image will be built as: "
+	@echo "$(APP_NAME):$(APP_VERSION)"
+	@echo "Docker image will be pushed as:"
+	@echo "$(DOCKER_REPO):$(APP_VERSION)"
+
+.PHONY: docker-build-base
+docker-build-base:							## build the base docker image
+	docker build "$(APP_ROOT)" -f "$(APP_ROOT)/docker/Dockerfile-base" -t "$(APP_NAME):base"
+	docker tag "$(APP_NAME):base" "$(APP_NAME):latest"
+	docker tag "$(APP_NAME):base" "$(DOCKER_REPO):latest"
+	docker tag "$(APP_NAME):base" "$(DOCKER_REPO):$(APP_VERSION)"
+
+.PHONY: docker-build-webservice
+docker-build-webservice: docker-build-base		## build the web service docker image
+	docker build "$(APP_ROOT)" -f "$(APP_ROOT)/docker/Dockerfile-webservice" -t "$(APP_NAME):$(APP_VERSION)-webservice"
+	docker tag "$(APP_NAME):$(APP_VERSION)-webservice" "$(APP_NAME):latest-webservice"
+	docker tag "$(APP_NAME):$(APP_VERSION)-webservice" "$(DOCKER_REPO):latest-webservice"
+	docker tag "$(APP_NAME):$(APP_VERSION)-webservice" "$(DOCKER_REPO):$(APP_VERSION)-webservice"
+
+.PHONY: docker-build-worker
+docker-build-worker: docker-build-base		## build the worker docker image
+	docker build "$(APP_ROOT)" -f "$(APP_ROOT)/docker/Dockerfile-worker" -t "$(APP_NAME):$(APP_VERSION)-worker"
+	docker tag "$(APP_NAME):$(APP_VERSION)-worker" "$(APP_NAME):latest-worker"
+	docker tag "$(APP_NAME):$(APP_VERSION)-worker" "$(DOCKER_REPO):latest-worker"
+	docker tag "$(APP_NAME):$(APP_VERSION)-worker" "$(DOCKER_REPO):$(APP_VERSION)-worker"
 
 .PHONY: docker-build
-docker-build:	## build Docker image
-	docker build "$(APP_ROOT)" -t "$(LATEST_TAG)"
-	docker tag "$(LATEST_TAG)" "$(DOCKER_TAG)"
+docker-build: docker-build-base docker-build-webservice docker-build-worker		## build all docker images
 
 .PHONY: docker-push
 docker-push: docker-build	## push built docker image
 	docker push "$(DOCKER_TAG)"
```

Comment:
should probably be modified to a simple target that inherits from all the other `docker-push-...` to push everything at once

Reply:
In fact it was a duplicate to the better target here : https://github.com/Ouranosinc/cowbird/blob/request-queue/Makefile#L368
I will remove it

Reply:
I think it is valid to have that target. It should just be as such so it runs all of them: 

```
docker-push: docker-push-webservice docker-push-worker
```

Reply:
Exact, but the target docker-push already exist and do what you want, so I removed the other one.